09_CalculatorStrategyCoR/src/main/java/bootcamp/process/element/impl/DividerFactory.java

```diff
@@ -0,0 +1,19 @@
+package bootcamp.process.element.impl;
+
+import bootcamp.process.element.ElementFactory;
+import bootcamp.process.element.ProcessingElement;
+
+public class DividerFactory extends ElementFactory {
+    public DividerFactory(ElementFactory nextFactory, String operator) {
```

Comment:
What do you think about having an Enum for operators?  so that you can avoid unexpected operators when calling this constructor or the methods inside it :)